api/src/pcapi/admin/custom_views/custom_reimbursement_rule_view.py

```diff
@@ -153,7 +171,13 @@ def format_venue(view, context, model, name):
     if model.offererId:
         return None
     venue = model.offer.venue
-    return venue.publicName or venue.name
+    humanized_offerer_id = human_ids.humanize(venue.managingOffererId)
+    humanized_venue_id = human_ids.humanize(venue.id)
+    url = f"{settings.PRO_URL}/structures/{humanized_offerer_id}/lieux/{humanized_venue_id}"
```

Comment:
```suggestion
    if venue.isVirtual:
        url = f"{settings.PRO_URL}/accueil?structure={humanize(offerer_id)}"
    else:
        f"{settings.PRO_URL}/structures/{humanize(offerer_id)}/lieux/{humanize(venue_id)}"
```
Il me semble que l'on doive faire une petite pirouette ici.

Reply:
bien vu, je corrige. Merci !